composer.json

```diff
@@ -84,28 +84,20 @@
     }
   },
   "scripts": {
-    "post-install-cmd": [
-      "@auto-scripts"
-    ],
-    "post-update-cmd": [
-      "@auto-scripts"
-    ],
-    "analyze": "@php ./vendor/bin/phpstan",
-    "auto-scripts": [
-      "[ $COMPOSER_DEV_MODE -eq 0 ] || composer normalize"
-    ],
-    "cs": "@php ./vendor/bin/ecs check",
-    "cs:fix": "@php ./vendor/bin/ecs check --fix",
-    "grump": "php ./vendor/bin/grumphp run",
-    "grump:install": "php ./vendor/bin/grumphp git:init",
-    "lint": "@php ./vendor/bin/parallel-lint --exclude .git --exclude vendor .",
+    "analyze": "phpstan",
+    "cs": "ecs check",
+    "cs:fix": "ecs check --fix",
+    "grump": "grumphp run",
+    "grump:install": "grumphp git:init",
+    "lint": "parallel-lint --exclude .git --exclude vendor .",
     "qa": [
+      "@composer normalize --dry-run",
```

Comment:
Excuse me. This is a logical contradiction. `normalize` **is** the fixer.

Reply:
I leave this 100% up to you.

Reply:
I use `composer-normalize` since quite some time. This is not about learning composer normalize arguments.

From that point of view, you can remove almost everything since `composer test` is just `./vendor/bin/phpunit`, etc.

It's just about having consistent commands set (which means almost guessable). `composer cs` checks coding standards, `composer cs:fix` fixes coding standards, and so on. From that POV, using a `--dry-run` argument feels like an anomaly, a different behavior from other commands. 

This a just a legit debate IMO, I didn't mean to annoy you.

Reply:
I see you like organize books on a bookshelf.
Please do not store a cookbook there.
It is called `composer normalize`. We don't need to rename it!!

`composer config:check`
`composer config:fix`
These would be the Laravel `artisan` equivalents.

Reply:
> I see you like organizing books on a bookshelf.

My mother used to keep her cookbooks in the oven.

Reply:
Thanks for your valuable and constructive input.

I renamed the script `lint-composer` to avoid confusion with `composer normalize`.